src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -150,6 +153,7 @@ def __init__(self, *args, **kwargs):
         super(TestWithServers, self).__init__(*args, **kwargs)
 
         self.agent_sessions = None
+        self.setup_start_servers_and_clients = True
```

Comment:
Based upon the next comment, this should probably be renamed to something like `self.setup_start_servers`